backend/src/config/index.ts

```diff
@@ -12,6 +12,7 @@ Decimal.set({
 const constants = {
   DB_VERSION: '0035-admin_pending_creations_decimal',
   DECAY_START_TIME: new Date('2021-05-13 17:46:31'), // GMT+0
+  LOG4JS_CONFIG: 'log4js-config.json',
   CONFIG_VERSION: {
     DEFAULT: 'DEFAULT',
     EXPECTED: 'v6.2022-04-21',
```

Comment:
This must be updated. We try to ensure this way, that the `.env` on production is updated manually with the new version